client/app/widgets/Resources/index.jsx

```diff
@@ -3,25 +3,84 @@ import React from 'react';
 import css from './Resources.scss';
 import { Resource } from '../../components/Resource';
 import { Utils } from '../../utils';
+import { InputTag } from '../../components/Input/InputTag';
 
 export type Props = {
   resources: any,
 };
 
-export const Resources = (props: Props) => {
-  const { resources } = props;
-  return (
-    <div className={css.gridThree}>
-      {resources.map(resource => (
-        <div className={css.gridThreeItem} key={Utils.randomString()}>
-          <Resource
-            tagged
-            tags={resource.languages.concat(resource.tags)}
-            title={resource.name}
-            link={resource.link}
+export type State = {
+  checkboxes: any
+};
+
+export type Selected = {
+  checked: boolean, 
+  id: string, 
+  key: string, 
+  label: string, 
+  value: string
+}
+
+export class Resources extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = { checkboxes: this.createCheckboxes() };
+  }
+  
+  createCheckboxes = () => {
+    const { resources } = this.props;
+    const tagsList = [...new Set(resources.map(res => res.tags.concat(res.languages)).reduce((acc, val) => acc.concat(val), []))];
+    return tagsList.map<*>((tag: string) => {return {id: tag, key: tag, value: tag, label: tag, checked: false}});
+  };
+  
+  checkboxChange = (box: Selected) => {
+    this.setState((prevState: State) => {
+      const updatedBoxes = prevState.checkboxes.filter(checkbox => checkbox.id !== box.id).concat(box);
+      return { checkboxes: updatedBoxes };
+    });
+  };
+  
+  filterList = (check: Array<*>) => {
+    const { resources } = this.props;
+    const selectedTags = check.filter(c => c.checked === true);
+    const matchingResources = resources.filter(r => {
+      const tagCheck = selectedTags.map((t: Selected)  => 
+        r.tags.concat(r.languages).includes(t.id)
+      );
+      if(tagCheck.includes(false)) {
+        return false;
+      } else {
+        return true;
+      } 
+    });
+    return matchingResources;
+  };
+  
+  render() {
+    const { checkboxes } = this.state;
+    const filteredResources = this.filterList(checkboxes);
+    return (
+      <React.Fragment>
+          <InputTag
+            id='resourceTags'
+            name='resourceTags'
+            placeholder='Press ENTER to add'
```

Comment:
We can use i18n here, look at other components in client for an example of how to do this.